web/src/portico/hello.js

```diff
@@ -0,0 +1,56 @@
+import $ from "jquery";
+
+function get_new_rand(oldRand, max) {
+    const newRand = Math.floor(Math.random() * max);
+    return newRand === oldRand ? get_new_rand(newRand, max) : newRand;
+}
+
+function get_random_item_from_array(array) {
+    return array[Math.floor(Math.random() * array.length)];
+}
+
+const current_clint_logo_class_names = new Set([
+    "client-logos__logo_akamai",
+    "client-logos__logo_tum",
+    "client-logos__logo_wikimedia",
+    "client-logos__logo_rust",
+    "client-logos__logo_dr_on_demand",
+    "client-logos__logo_maria",
+]);
+const future_logo_class_names = new Set([
```

Comment:
And for parallelism with the above:
```suggestion
const future_client_logo_class_names = new Set([
```